### DISCUSSION TITLE
How to cancel a QR code after I have scanned it with the Go Pro Hero 10

### DISCUSSION THREAD

**Discussion by ntentoma** [gopro/labs#309] [Q&A]
Hello,

I was wondering if anyone knows, how to cancel QR code\action after I scanned it with my Go Pro Hero 10. For example , let's say I create a QR Code:

!r-3900NmTp.5r4fW0tw55i4M1x-.5g1v1q1!S!7200E

and after I scanned it, I decide I don't  want to use it. How can I cancel it? 

Thank you,

Nick

**Comment by dnewman-gpsw** [comment]
Turn on the camera, and press a button when is askes if you want to cancel.

**Reply by ntentoma** [reply]
Thank you for the reply,

The message was staying on my screen for a very short period of time and it was hard to read. Eventually,  after many tries i was able to cancel it. Also my screen was shutting down after, i think, 20-30 sec from the time it had started.
